internal/ini/fuzz_test.go

```diff
@@ -1,29 +1,57 @@
-// +build fuzz
+// +build gofuzzbeta
 
-// fuzz test data is stored in Amazon S3.
-package ini_test
+package ini
 
 import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
 	"path/filepath"
 	"testing"
-
-	"github.com/aws/aws-sdk-go-v2/internal/ini"
 )
 
-// TestFuzz is used to test for crashes and not validity of the input
-func TestFuzz(t *testing.T) {
-	paths, err := filepath.Glob("testdata/fuzz/*")
+func FuzzParse(f *testing.F) {
+	corpus, err := loadCorpus(
+		// Also loads "testdata/corpus/FuzzParse" automatically,
+		// for previous known panics if present and contains any files
+		filepath.Join("testdata", "valid", "*"),
+		filepath.Join("testdata", "invalid", "*"),
+	)
```

Comment:
How are these files used? Do we need to prefill cases which are run randomly?

Reply:
This reuses the existing tests that the rest of the ini parser's unit tests use.Theese are used to "seed" the corpus. It seems better to reuse these existing tests instead of duplicating them into the `testdata/corpus/FuzzParse`. folder.